packages/merkle-distributor/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@uma/merkle-distributor",
+  "version": "1.0.0",
+  "description": "Simple script for dealing with Merkle distribution contracts and proofs hosted on IPFS",
+  "dependencies": {
+    "@types/commander": "^2.12.2",
+    "@uma/core": "^2.0.1",
+    "chai": "^4.3.0",
+    "commander": "^7.1.0",
+    "ethers": "^5.0.31",
+    "ipfs-http-client": "^49.0.2",
+    "mocha": "^8.3.0",
+    "node-fetch": "^2.6.1",
+    "typescript": "^4.2.2"
+  },
+  "devDependencies": {
+    "web3": "1.2.7"
+  },
+  "homepage": "https://umaproject.org",
+  "license": "AGPL-3.0-or-later",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.com/",
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/UMAprotocol/protocol.git"
+  },
+  "bin": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "main": "dist/index.js",
+  "files": [
+    "/dist/**/*"
+  ],
+  "scripts": {
+    "build": "tsc -b"
```

Comment:
nit: related to the above, if you have a test, we should probably add a test script here.

Reply:
I added a script for the `integration-test`. I dont think we want to run this in CI imho.